src/legacy/core_plugins/expressions/public/np_ready/public/execute.ts

```diff
@@ -69,14 +74,14 @@ export class ExpressionDataHandler {
     this.abortController.abort();
   };
 
-  getData = async () => {
+  getData = async (): Promise<IInterpreterResult | IInterpreterErrorResult> => {
     try {
       return await this.promise;
     } catch (e) {
       return {
         type: 'error',
         error: {
-          type: e.type,
+          name: e.type,
```

Comment:
I think I prefer `type` to `name`, since it's really the type of error that is being extracted here.